test/unit/math/rev/err/check_positive_test.cpp

```diff
@@ -1,10 +1,44 @@
+#include <stan/math/rev/arr.hpp>
+#include <stan/math/rev/mat.hpp>
 #include <stan/math/rev/scal.hpp>
 #include <gtest/gtest.h>
+#include <vector>
 #include <exception>
 #include <limits>
 
 using stan::math::var;
 
+TEST(AgradRevErrorHandlingArray, CheckPositive) {
+  using stan::math::check_positive;
```

Comment:
Declare this outside just once out of test block.

Reply:
See above.